pkg/tests/ossm/smoke_test.go

```diff
@@ -63,6 +126,38 @@ func TestBookinfoInjection(t *testing.T) {
 	})
 }
 
+func assertProxiesReadyInLessThan10Seconds(t TestHelper, ns string) {
+	t.Log("Extracting proxy startup time and last transition time for all the pods in the namespace")
+	proxyTimeList := oc.GetJson(t, ns, "pods", "", `{range .items[*]}{.metadata.name}{"\t"}{.status.containerStatuses[?(@.name=="istio-proxy")].state.running.startedAt}{"\t"}{.status.conditions[?(@.type=="Ready")].lastTransitionTime}{"\n"}{end}`)
```

Comment:
This command is tricky and the parser below is even much trickier. I had to execute this command and see the output to understand what it does. What do you think about something like this?
```suggestion
	startedAt := oc.GetJson(t, ns, "pods", "", `{range .items[*]}{.status.containerStatuses[?(@.name=="istio-proxy")].state.running.startedAt}{end}`)
	readyAt := oc.GetJson(t, ns, "pods", "", `{range .items[*]}{.status.conditions[?(@.type=="Ready")].lastTransitionTime}{end}`)
	startedAt = time.Parse(...)
	readyAt = time.Parse(...)
	startupTime := readyAt.Sub(startedAt)
	if startupTime > 10*time.Second {
        	t.Fatalf("Proxy startup time is too long: %s", startupTime.String())
	}
```

Reply:
Sure, better in that way. More easy to read